app/src/services/auth.ts

```diff
@@ -39,7 +39,7 @@ function setTokens({ access_token, refresh_token }: Cookies) {
   setApiClientAuthToken(access_token);
 }
 
-async function authenticate({ body, url }: { body: string; url: string }) {
+export async function authenticate({ body, url }: { body: string; url: string }) {
```

Comment:
Why do we need to export it?

Reply:
It was a leftover from past work, good catch!